pkg/apis/kubermatic/v1/cluster.go

```diff
@@ -1310,6 +1310,11 @@ type OpenstackCloudSpec struct {
 	// The suffix is set to `nip.io` by default. Can only be used with the external CCM and might be deprecated and removed in
 	// future versions as it is considered a workaround only.
 	IngressHostnameSuffix *string `json:"ingressHostnameSuffix,omitempty"`
+
+	// Flag to configure enablement of CSI Cinder plugin for topology support.
```

Comment:
Nit: I think the comment might make more sense the other way
```suggestion
	// Flag to configure enablement of topology support for the Cinder CSI plugin.
```